gobblin-iceberg/src/test/java/org/apache/gobblin/iceberg/writer/HiveMetadataWriterTest.java

```diff
@@ -374,6 +374,7 @@ public void  testUpdateLatestSchemaWithExistingSchema() throws IOException {
 
 
     // Tables part of deny list, schema only fetched from hive on the first time and the all future calls will use the cache
+    Assert.assertTrue(updateLatestSchema.apply(tableNameDenied));
```

Comment:
Fat finger removed this line on accident, causing the UT to fail in previous commit